common/src/Model/LayerNode.cpp

```diff
@@ -91,6 +91,18 @@ namespace TrenchBroom {
             addOrUpdateAttribute(AttributeNames::LayerColor, color.toString());
         }
 
+        bool LayerNode::omitFromExport() const {
+            return hasAttribute(AttributeNames::LayerOmitFromExport, AttributeValues::LayerOmitFromExportValue);
+        }
+
+        void LayerNode::setOmitFromExport(const bool i_omitFromExport) {
```

Comment:
Why the `i_` prefix?

Reply:
Without the `i_` the variable name shadows the getter function with the same name and correctly produces a warning.
Not sure what the `i_` stands for (input?), but I thought this was our coding style for avoiding variable shadowing.

Reply:
Hmm - using `const bool omitFromExport` here used to be reported by `-Wshadow`, but it looks like they relaxed the warning in gcc 5 to not warn for this case. Still, I'd rather never shadow anything.

Reply:
Ah ok, I guess we have this type of shadowing all over the place, whenever we have getters, so I guess it's not a big deal and I can just remove the `i_`